ingestion/setup.py

```diff
@@ -33,7 +33,7 @@ def get_long_description():
     "boto3": "boto3>=1.20,<2.0",  # No need to add botocore separately. It's a dep from boto3
     "geoalchemy2": "GeoAlchemy2~=0.12",
     "google-cloud-storage": "google-cloud-storage==1.43.0",
-    "great-expectations": "great-expectations~=0.16.0",
+    "great-expectations": "great-expectations>=0.17.0",
```

Comment:
@TeddyCr should we keep this as `~=0.17`?

Reply:
Correct. cc @VolkovGeoPhy

Reply:
done